apps/omg_utils/test/omg_utils/http_rpc/validators/base_test.exs

```diff
@@ -160,53 +177,87 @@ defmodule OMG.Utils.HttpRPC.Validator.BaseTest do
              do: {:ok, %{currency: currency, amount: amount}}
       end
 
-      assert {:ok, %{currency: @bin_value, amount: 100}} =
+      {:ok, _address_value} = @params |> Map.get("valid_address") |> Encoding.from_hex()
+
+      assert {:ok, %{currency: address_value, amount: 100}} =
```

Comment:
you need to pin address_value

Reply:
look at the difference between this assertion and the one below:
```
assert {:ok, address_value} == expect(@params, "valid_address", :address)
```
`==`

Reply:
Thanks 🙏